packages/protocol-parser/src/schemaAbiTypes.ts

```diff
@@ -0,0 +1,211 @@
+import { AbiType } from "abitype";
```

Comment:
happy to move em over! It was just cleaner while prototyping to start these here from a clean slate

we should probably decide: do we like packages with multiple languages (i.e. schema-type having both solidity and TS) or should we split these/colocate them with their language-specific library (e.g. store and protocol-parser)

Reply:
if you don't mind, I'd prefer to move it over in a follow up as part of a larger refactor, because once I move it over and align naming, its likely a bunch of other downstream packages (e.g. network) will need updating too, and that feels outside the scope of this PR

Reply:
moved schema type stuff to schema-type in https://github.com/latticexyz/mud/pull/1029